streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -160,18 +162,21 @@ public void postCommit() {
 
     @Override
     public void closeClean() {
+        streamsMetrics.removeAllTaskLevelSensors(Thread.currentThread().getName(), id.toString());
```

Comment:
Standby tasks don't currently register any sensors, but I personally rather to be defensive and idempotently ensure we remove any sensors while closing.

Reply:
Sounds good. But why do it both here and in `closeDirty` vs doing so in `close(clean)`?

Reply:
I'd like to inline `close(boolean)`, but am resisting the urge... This is a compromise ;)

Reply:
Fair enough. I thought the answer might be something like that...